### ISSUE TITLE
Typedoc function overloads don't generate correctly

### ISSUE THREAD

**Issue by hawkerm** [hez2010/TypedocConverter#2]
For instance: https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.itextmodel.html#findmatches

Generates a single function with parameters for both overloads instead of two separate function signature declarations.

```csharp
        /// <summary>
        /// Search the model.
        /// </summary>
        /// <returns>
        /// The ranges where the matches are. It is empty if not matches have been found.
        /// 
        /// </returns>
        /// <summary>
        /// Search the model.
        /// </summary>
        /// <returns>
        /// The ranges where the matches are. It is empty if no matches have been found.
        /// 
        /// </returns>
        IAsyncOperation<FindMatch[]> FindMatches(string searchString, bool searchOnlyEditableRange, bool isRegex, bool matchCase, string wordSeparators, bool captureMatches, double limitResultCount, string searchString1, IRange searchScope, bool isRegex1, bool matchCase1, string wordSeparators1, bool captureMatches1, double limitResultCount1);        
```

instead of

```csharp
        /// <summary>
        /// Search the model.
        /// </summary>
        /// <returns>
        /// The ranges where the matches are. It is empty if not matches have been found.
        /// 
        /// </returns>
        IAsyncOperation<FindMatch[]> FindMatches(string searchString, bool searchOnlyEditableRange, bool isRegex, bool matchCase, string wordSeparators, bool captureMatches, double limitResultCount);

        /// <summary>
        /// Search the model.
        /// </summary>
        /// <returns>
        /// The ranges where the matches are. It is empty if no matches have been found.
        /// 
        /// </returns>
        IAsyncOperation<FindMatch[]> FindMatches(string searchString, IRange searchScope, bool isRegex, bool matchCase, string wordSeparators, bool captureMatches, double limitResultCount);
```

**Comment by hawkerm** [comment]
I had already edited some of the types, but you should see the same behavior from the generation.

**Comment by hez2010** [comment]
Fixed via c7965a15ca46d5dad2a9804dc3a0f4b4a6d1dcd5
